### ISSUE TITLE
Override "Save draft" button's default value

### ISSUE THREAD

**Issue by ladrua** [wagtail/wagtail#4282]
Should be a way to set "Save draft" buttton to other value by default.
In a lot of simpler projects with one user/admin its most common to just publish directly.

**Comment by capiscuas** [comment]
Yes, this would be really useful, since the Save draft sometimes is confusing for users that just want to send to moderation or publish directly.

**Comment by germn** [comment]
I posted on SO [a temporary solution](https://stackoverflow.com/a/55438103/1113207) for client side (Tampermonkey based). It does job until relevant hook isn't implemented on Wagtail side.

**Comment by brylie** [comment]
We would also like to set the default action to "Publish" on particular content types that don't need review/moderation. This is to prevent the repeated mistake where the content editor forgets to select "publish" and inadvertently saves a draft.

**Comment by zerolab** [comment]
You can achieve this using [`construct_page_action_menu`](https://docs.wagtail.org/en/stable/reference/hooks.html#construct-page-action-menu)

And you can define your own with [`register_page_action_menu_item`](https://docs.wagtail.org/en/stable/reference/hooks.html#construct-page-action-menu)

Given that, I'd be inclined to close this issue 🤔

**Comment by brylie** [comment]
@zerolab, the `construct_page_action_menu` seems to be a global hook that would affect all `Page` models. Is there a way to set the default action to "publish" for a particular `Page` models?

**Comment by zerolab** [comment]
@brylie the `context.get("page")` should give you what you need
